doop.c

```diff
@@ -657,17 +688,72 @@ Magic and tainting are handled.
 =cut
 */
 
+static SV *
+S_concat_maybe_overloaded(pTHX_ SV *lhs, SV *rhs)
```

Comment:
this should be in embed.fnc, and should include the appropriate PERL_ARGS_ macros.

Reply:
Also needs pod.